packages/website/docs/helpers.docsearch.md

```diff
@@ -0,0 +1,240 @@
+---
+title: helpers.docsearch
+---
+
+:::info
+
+The following content is for **[DocSearch v3][2]** and **[its new infrastructure][5]**. If you are using **[DocSearch v2][3]** or the **[docsearch-scraper][6]**, see the **[legacy][4]** documentation.
+
+:::
+
+## Introduction
+
+:::note
+
+This documentation will only contain informations regarding the **helpers.docsearch** method, see **[Algolia Crawler Documentation][7]** for more informations on the **[Algolia Crawler][8]**.
```

Comment:
This might be good to document it in our main Doc too, as we are now exposing it to everyone